samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ItemAddListener.java

```diff
@@ -1124,25 +1124,23 @@ else if (itemauthor.getTarget() != null && (itemauthor.getTarget().equals("samba
         }
 
         QuestionPoolService qpdelegate = new QuestionPoolService();
-	// removed the old pool-item mappings
-          if ( (bean.getOrigPool() != null) && (!bean.getOrigPool().equals(""))) {
-            qpdelegate.removeQuestionFromPool(item.getItemId(),
-            		Long.valueOf(bean.getOrigPool()));
-          }
+        // removed the old pool-item mappings
+        if (StringUtils.isNotEmpty(bean.getOrigPool())
```

Comment:
i think you want isNotBlank here as whitespace is obv not valid

Reply:
I'm just replacing like for like, but I can change it. It was a deliberate choice though.